tests/test_port/fixtures/tables.md

```diff
@@ -273,106 +273,6 @@ bar|bar
 .
 
 
-Should be terminated via row without "|" symbol:
```

Comment:
I think it would be best to leave these tests in and, if necessary, change the expected outcomes and maybe the heading.
Just so it's clear what the new behaviour is

Reply:
:+1: added back the tests